chromium_src/courgette/third_party/bsdiff/paged_array.h

```diff
@@ -0,0 +1,27 @@
+#ifndef BRAVE_CHROMIUM_SRC_COURGETTE_THIRD_PARTY_BSDIFF_PAGED_ARRAY_H_
```

Comment:
please add a copyright (with 2022 year) and clang-format everything (seems like there are two empty lines below).